examples/camera-app/camera-common/include/camera-app.h

```diff
@@ -32,18 +32,19 @@ class CameraApp
 
 public:
     // This class is responsible for initialising all the camera clusters and managing the interactions between them
-    explicit CameraApp(chip::EndpointId aClustersEndpoint, CameraDeviceInterface * cameraDevice) :
-        mChimeServer(aClustersEndpoint, cameraDevice->GetChimeDelegate()),
-        mWebRTCTransportProvider(cameraDevice->GetWebRTCProviderDelegate(), aClustersEndpoint)
-    {}
+    CameraApp(chip::EndpointId aClustersEndpoint, CameraDeviceInterface * cameraDevice);
 
     // Initialize all the camera device clusters.
     void InitCameraDeviceClusters();
 
 private:
+    chip::EndpointId mEndpoint;
+    CameraDeviceInterface * mCameraDevice;
+
     // SDK cluster servers
-    chip::app::Clusters::ChimeServer mChimeServer;
-    chip::app::Clusters::WebRTCTransportProvider::WebRTCTransportProviderServer mWebRTCTransportProvider;
+    std::unique_ptr<chip::app::Clusters::WebRTCTransportProvider::WebRTCTransportProviderServer> mWebRTCTransportProviderPtr;
```

Comment:
We already use unique_ptr 'Platform::UniquePtr<CameraApp> gCameraApp' to manage CameraApp, no need to use unique_ptr to manage each class member varible.